### ISSUE TITLE
Feature to append new panes to existing window.

### ISSUE THREAD

**Issue by greymd** [greymd/tmux-xpanes#79]
I want the feature like this. `-a` means `append`.
Or, `-x` (`extend`) may also be meaningful.

```
    +-------------------------------+-------------------------------+
    │$                              │$                              │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    +-------------------------------+-------------------------------+
    │ $ xpanes -a 4 5 6                                             │
    │                                                               │
    │                                                               │
    │                                                               │
    │                                                               │
    │                                                               │
    │                                                               │
    │                                                               │
    +-------------------------------+-------------------------------+
```

**Result:**

```
    +-------------------------------+-------------------------------+
    │$                              │$                              │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    +-------------------------------+-------------------------------+
    │$                              │$ echo 4                       │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    +-------------------------------+-------------------------------+
    │$ echo 5                       │$ echo 6                       │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    │                               │                               │
    +-------------------------------+-------------------------------+
```

**Comment by greymd** [comment]
* If the pane is already synchronized, show warning message.
* Automatically disable the synchronization after the separation.
This feature can be used with existing feature.
But all the features are not valid.

```
pipe-mode    : valid
--stay       : ignore or warning message.
-S           : ignore or warning message.
-d           : ignore
--log        : valid
--log-format : valid
-l           : valid
-I           : valid
-e           : valid
-c           : valid
-V           : valid
-h           : valid
-t           : valid (New window gets title also.)
```

**Comment by greymd** [comment]
I suddenly thought that `-x` is cooler than `-a`.

**Comment by greymd** [comment]
Released as `-x` option in v3.0.0
https://github.com/greymd/tmux-xpanes/releases
